stub-packages/merge-styles/src/index.ts

```diff
@@ -0,0 +1,2 @@
+import './version';
```

Comment:
merge-styles in v8 currently exports the following files
fontFace
IRawStyleBase
IStyle
keyframes
mergeStyles
mergeStyleSets
server"